### PR TITLE
Fixed FluidMaterial crashing the game if GENERATE_FLUID_BLOCK and GENERATE_PLASMA are present

### PR DESCRIPTION
**Pull request by idcppl**
**What:**
fixed issue #1358 
It was trying to generate the normal version and plasma version with the same name.

### CODE REVIEW

File: src/main/java/gregtech/common/MetaFluids.java [hunk 1]
```diff
@@ -163,7 +163,7 @@ private static void setAlternativeFluidName(FluidMaterial material, FluidType fl
         alternativeFluidNames.put(fluidType.getFluidName(material), alternativeName);
     }
 
-    public static Fluid registerFluid(FluidMaterial material, FluidType fluidType, int temperature) {
+    public static Fluid registerFluid(FluidMaterial material, FluidType fluidType, int temperature, boolean isPlasma) {
```
Comment (LAGIdiot) [inline src/main/java/gregtech/common/MetaFluids.java:166]: You don't need to pass in additional param as you can already get that information from `fluidType`

**Review by LAGIdiot — COMMENTED** [review]
I am not sure how we want to handle this so please wait for direction from related bug.

**Comment by idcppl** [comment]
Fixed it, didn't even think about that.

**Comment by LAGIdiot** [comment]
Please update this PR regarding conclusion from connected bug.

You should fix you opening comment to get GitHub correctly link bug that this fixes.
And also please make commit comments descriptive in regard to what they change and don't put useless text to them (LUL).

**Comment by LAGIdiot** [comment]
Are you still working on this PR? Do you need help with anything?

**Comment by idcppl** [comment]
I don't know where you want actually want to go with this, kind of made the PR before actually looking at how to tackle it through suggestions. I'm just going to close this and leave the bug to someone else.

**Pull request closed without merging**